.github/workflows/release.yaml

```diff
@@ -0,0 +1,61 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+      # TODO: Should be scoped to tag pushes eventually
+
+jobs:
+  docker_release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: ngrok/ngrok-ingress-controller:latest
+    
+  helm_release:
+    needs: docker_release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
whats up here? is this required?

Reply:
yea ... it wasn't finding the chart without it.  This took way too much time to figure out, and thankfully I then found [this comment](https://giters.com/helm/chart-releaser-action/issues/69#issuecomment-730622120).